.gitignore

```diff
@@ -1,3 +1,4 @@
 composer.phar
 composer.lock
 vendor
+.idea
```

Comment:
Please maintain IDE-specific ignores in [your global gitignore](https://git-scm.com/docs/gitignore) or [your working copy's local exclude file](https://help.github.com/en/articles/ignoring-files#explicit-repository-excludes).